media/js/app/taxonomy/taxonomy.js

```diff
@@ -1,19 +1,19 @@
 (function (jQuery) {
-    
+
     Term = Backbone.Model.extend({
         urlRoot: '/api/term/',
-        toTemplate: function() {
+        toTemplate: function () {
             return _(this.attributes).clone();
-        }        
-    });    
+        }
+    });
 
     var TermList = Backbone.Collection.extend({
         urlRoot: '/api/term/',
         model: Term,
-        comparator: function(obj) {
+        comparator: function (obj) {
```

Comment:
A minor note: Our team here tries to adhere to community standards for our python and javascript code. (See here http://javascript.crockford.com/code.html). The standard here is not to follow the "function" declaration here with a space.